.github/workflows/spring-oracle-example.yml

```
@@ -0,0 +1,68 @@
+name: spring-oracle-example CI Build
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)